packages/knip/src/plugins/nx/index.ts

```diff
@@ -28,9 +28,11 @@ const findNxDependenciesInNxJson: GenericPluginCallback = async configFilePath =
 
   const plugins =
     localConfig.plugins && Array.isArray(localConfig.plugins)
-      ? localConfig.plugins.map(it => getPackageNameFromModuleSpecifier(it.plugin)).filter(value => value !== undefined)
+      ? localConfig.plugins
+          .map(item => (typeof item === 'string' ? item : item.plugin))
+          .map(it => getPackageNameFromModuleSpecifier(it))
+          .filter(value => value !== undefined)
```

Comment:
Nitpick: I usually skip this filter if there's a `compact()` on the final result (filtering out falsy values and duplicates).

Reply:
Noted for next time! Thanks